raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -283,3 +282,78 @@ def get_global_causal_policy(self, post_data):
                     "inner error: {}".format(e_str),
                 WidgetRequestResponseConstants.data: []
             }
+
+    def forecast(self, post_data):
+        try:
+            filters = post_data[0]
+            # do we need this? ask ilya
```

Comment:
@imatiach-msft ? Any idea what is meant here?

Taking a step back, I would be reluctant to allow further filtering beyond the time series. Filtering to filter out entire time series, sure, but just showing a subset of points from one time series seems strange (?) Wdyt?